src/ActivityLogger.php

```diff
@@ -91,6 +91,17 @@ public function byAnonymous()
         return $this->causedByAnonymous();
     }
 
+    public function event(?string $event)
+    {
+        return $this->setEvent($event);
```

Comment:
Is there any reason to allow `null` as event? 🤔 
Right now we don't allow `null` for any method in the activity builder.